### ISSUE TITLE
ci: enable gh actions with arm64

### ISSUE THREAD

**Issue by glimchb** [jaypipes/ghw#334]
Today with #303 we doing snapshot based testing

We discussed in several PRs and Issues how to enhance that and add real ci/cd with gh actions.

GH Actions doesn't support aarch64 today, unfortunately: 
- https://github.com/actions/runner-images/issues/5631#issuecomment-1218092275

So try to do QEMU maybe, although... see - https://github.com/actions/runner-images/issues/183

Another approach is to run 1 single ci job for ARM64 only on https://www.travis-ci.com/

**Comment by jaypipes** [comment]
> Another approach is to run 1 single ci job for ARM64 only on https://www.travis-ci.com/

Yeah, I've moved ghw off of Travis and strictly on to GH Actions about a year or so ago... I'd like to keep it that way for simplicity reasons. I will follow that ARM64 issue for runner-images.
